sdk/python/kubeflow/training/api/training_client_test.py

```diff
@@ -162,8 +162,21 @@ def create_job():
 
 def generate_job_with_status(
     job: constants.JOB_MODELS_TYPE,
+    namespace: str = constants.DEFAULT_NAMESPACE,
     condition_type: str = constants.JOB_CONDITION_SUCCEEDED,
 ) -> constants.JOB_MODELS_TYPE:
+
+    if namespace == FAILED:
+        condition_type = constants.JOB_CONDITION_FAILED
+    elif namespace == CREATED:
+        condition_type = constants.JOB_CONDITION_CREATED
+    elif namespace == RESTARTING:
+        condition_type = constants.JOB_CONDITION_RESTARTING
+    elif namespace == RUNNING:
+        condition_type = constants.JOB_CONDITION_RUNNING
+    elif namespace == SUCCEEDED:
+        condition_type = constants.JOB_CONDITION_SUCCEEDED
```

Comment:
Do you need this or you can just directly pass the required condition_type in generate_job_with_status function ?